### PR TITLE
Make jvm arguments more flexible

### PR DESCRIPTION
**Pull request by dcrissman**

### CODE REVIEW

**Comment by luan-cestari** [comment]
Nice. No problem to merge these changes. I thought that maybe we could rename the 'jvmOptions'  to something else as the options must start with 'X' and all 'X' options are for an specific jvm implementation ( http://www.oracle.com/technetwork/java/javase/tech/vmoptions-jsp-140102.html ). What do you think @dcrissman ?

**Comment by dcrissman** [comment]
Certainly, any ideas? We could simply make the leading X required to be passed in also if that would increase flexibility.

File: templates/application/migrator/lightblue-migrator-daemon.sh.erb [hunk 1]
```diff
@@ -45,7 +45,7 @@ do_exec()
   
   if @jvmOptions
     @jvmOptions.each do |key, value|
-      jsvcCommand << "-X#{key}=#{value}"
+      jsvcCommand << "-X#{key}#{value}"
```
Comment (luan-cestari) [inline templates/application/migrator/lightblue-migrator-daemon.sh.erb]: Noticed there isn't any space from one option to another , could you create a test to see if it work fine please? (maybe using Xms and Xmx, or can be any other parameter). Also, the jvmOptions might be used for some parameters that need '=' sign and other things (there are a lot of JVM parameters)

**Comment by dcrissman** [comment]
@luan-cestari and I chatted, and I will change it so that the entire option has to be entered by the user. So something equivalent to ["-Xmx###","-Xms###"]

**Pull request merged**